config/controller/kustomization.yaml

```diff
@@ -6,4 +6,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: public.ecr.aws/aws-controllers-k8s/iam-controller
-  newTag: 1.3.9
+  newTag: 1.3.8
```

Comment:
You need to pull the controller latest tags (git pull --tags upstream) and regenerate